Assets/Scripts/TankController.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+
+public class TankController
+{
+    public TankController(TankModel tankModel, TankView tankPrefab)
+    {
+        TankModel = tankModel;
+        TankView = GameObject.Instantiate<TankView>(tankPrefab);
+    }
+
+    public TankModel TankModel { get; }
+    public TankView TankView { get; }
+
```

Comment:
Add Method summary so that everybody get to know what exactly the method is doing.